CheckDrive.Web/CheckDrive.Web/Views/Shared/_SideBar.cshtml

```diff
@@ -71,32 +88,13 @@
              target=".dock-target" enableGestures="false">
     <e-content-template>
 
-        <div class="logo-wrapper">
-            <div class="app-logo">
-                <img src='~/images/LogoCheakDrive.png' style="height: 50px; width:50px" />
-            </div>
-            <div class="w-100"></div>
-            <i id="sidebar-toggler" class="fa-solid"></i>
-        </div>
-
-        <div class="profile-wrapper">
-            <i class="fa-solid fa-user"></i>
-            <div class="titles d-flex flex-column ps-3">
-                <h6 class="mb-0">@TempData["UserName"]</h6>
-            </div>
+        <div class="app-logo">
+            <img src='~/LogoCheakDrive.png' style="height: 80px; width:80px" />
```

Comment:
can we move this styling to **style.css**?

Reply:
fixed